packages/autorest.typescript/test/rlcIntegration/lroRest.spec.ts

```diff
@@ -995,7 +937,7 @@ describe("LRO Rest Client", () => {
         await poller.pollUntilDone();
         assert.fail("Scenario should throw");
       } catch (error) {
-        assert.equal(error.statusCode, 400);
+        assert.equal(error.message, "status.toLowerCase is not a function");
```

Comment:
@deyaaeldeen Could you refine the error message? The below one doesn't make sense from client side.

```
status.toLowerCase is not a function
```

Reply:
This has been addressed in https://github.com/Azure/azure-sdk-for-js/pull/23296. The following message is raised instead:
> Polling was unsuccessful. Expected status to have a string value or no value but it has instead: ${status}. This doesn't necessarily indicate the operation has failed. Check your Azure subscription or resource status for more information.